### PR TITLE
Update getSerializable in artifact.py

### PR DESCRIPTION
**Pull request by aakahlow**
This fixes the Jira issue: [DARCHR-208](https://gem5.atlassian.net/browse/DARCHR-208?focusedCommentId=10642&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-10642)

### CODE REVIEW

File: artifact/gem5art/artifact/artifact.py [hunk 1]
```diff
@@ -229,8 +229,12 @@ def __repr__(self) -> str:
         return vars(self).__repr__()
 
     def _getSerializable(self) -> Dict[str, Union[str, UUID]]:
-        data = vars(self)
+        data = vars(self).copy()
```
Comment (powerjg) [inline artifact/gem5art/artifact/artifact.py:232]: This seems like it was a latent bug! Good find!

File: artifact/gem5art/artifact/artifact.py [hunk 2]
```diff
@@ -229,8 +229,12 @@ def __repr__(self) -> str:
...
         data['inputs'] = [input._id for input in self.inputs]
+        if isinstance(data['cwd'], Path):
```
Comment (powerjg) [inline artifact/gem5art/artifact/artifact.py]: These `isinstance`s shouldn't be here. We *know* it's a `Path`, right?

Comment (aakahlow) [reply artifact/gem5art/artifact/artifact.py]: Yeah, this is redundant. Will update this

**Review by powerjg — APPROVED** [review]

**Comment by powerjg** [comment]
Bump to 1.0.1 and re-release!

Thanks!

**Pull request merged**